### DISCUSSION TITLE
Is there any way to change system tray icons?

### DISCUSSION THREAD

**Discussion by neomikus** [linuxmint/discussions#35] [Questions]
Hi, I'm looking to replicate a Windows XP aesthetic in Cinnamon, but despite changing the theme, some tray icons seem to be high-contrast still (see image), is there any way to customize them?

![Tray](https://github.com/linuxmint/discussions/assets/46509432/5a7770bf-b90e-4284-9f7f-030fb37dd44c)

Bonus points if you know how to customize third party apps tray icons, like Nvidia or Flameshot.

Thanks in advance!

**Comment by 100savage** [comment]
Place the Windows XP icons in the /usr/share/icons/ folder
`sudo cp -R "location/of/the/XP/icons" /usr/share/icons/`
Set the correct permissions to the XP Icons
sudo chmod -R 755 " /usr/share/icons/name/of/XP/folder"
